bucoffea/vbfhinv/vbfhinvProcessor.py

```diff
@@ -200,18 +201,13 @@ def process(self, df):
         # Set up ParticleNet
         pfcands = load_pf_cands(df,[muons,electrons])
         session = load_particlenet_model(bucoffea_path("particlenet_models/model_ops12.onnx"))
-        
-        # Remove jets in accordance with the noise recipe
-        if not cfg.RUN.ULEGACYV8 and df['year'] == 2017:
-            ak4   = ak4[(ak4.ptraw>50) | (ak4.abseta<2.65) | (ak4.abseta>3.139)]
-            bjets = bjets[(bjets.ptraw>50) | (bjets.abseta<2.65) | (bjets.abseta>3.139)]
 
         # Filtering ak4 jets according to pileup ID
         ak4 = ak4[ak4.puid]
 
         # Recalculate MET pt and phi based on npv-corrections
+        met_pt_uncorr, met_phi_uncorr = met_pt, met_phi
         if cfg.MET.XYCORR:
-            met_pt_uncorr, met_phi_uncorr = met_pt, met_phi
```

Comment:
These variables are used below (see [here](https://github.com/alpakpinar/bucoffea/blob/feature/particlenet/bucoffea/vbfhinv/vbfhinvProcessor.py#L247)) to compute uncorrected recoil (which is not really used a lot). So if we move these to `if` block, we probably need to adjust some code further down below (sorry if you already did this, maybe I've missed it)

Reply:
Done